src/foam/util/geo/GeolocationSupport.js

```diff
@@ -75,7 +75,8 @@ foam.CLASS({
         try {
           var ipStr = IPSupport.instance().getRemoteIp(x);
           var ip = InetAddress.getByName(ipStr);
-          File database = x.get(foam.nanos.fs.Storage.class).get("GeoLite2-City/GeoLite2-City.mmdb");
+          var jrlhome = System.getenv("JOURNAL_OUT");
+          File database = new File(jrlhome + "/GeoLite2-City/GeoLite2-City.mmdb");
```

Comment:
I wonder if we should the filesystem storage instead of juggling with JOURNAL_OUT env.
```
File database = x.get(Storage.class).get("GeoLite2-City/GeoLite2-City.mmdb");
```
@noodlemoodle @jlhughes @kgrgreer